data/repos.yml

```diff
@@ -1235,7 +1235,6 @@
 - repo_name: seal
   team: "#govuk-platform-security-reliability-team"
   management_url: https://dashboard.heroku.com/apps/govuk-seal
-  production_hosted_on: heroku
```

Comment:
Is there a github actions equivalent in Dev Docs config?

Reply:
(If not, should there be?)

Reply:
Note that we determine whether or not a repo is an "app" by [looking at `production_hosted_on`](https://github.com/alphagov/govuk-developer-docs/blob/cc8d6ef03b100f7a6e3187b8c71f3672dd7a9d04/app/repo.rb#L49-L51), so assuming we agree that it no longer counts as an "app", we probably want it to be `nil`.

Reply:
Fair 👍